deploy/aliyun/README-CN.md

```diff
@@ -3,8 +3,9 @@
 ## 环境需求
 
 - [aliyun-cli](https://github.com/aliyun/aliyun-cli) >= 3.0.15 并且[配置 aliyun-cli](https://www.alibabacloud.com/help/doc-detail/90766.htm?spm=a2c63.l28256.a3.4.7b52a893EFVglq)
+> **注意：** Access Key 需要具有操作相应资源的权限
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) >= 1.12
-- [helm](https://github.com/helm/helm/blob/master/docs/install.md#installing-the-helm-client) >= 2.9.1
+- [helm](https://github.com/helm/helm/blob/master/docs/install.md#installing-the-helm-client) >= 2.9.1 且 <= 2.11.0
```

Comment:
Why is the maximum version required?

Reply:
Compatible with tiller version.
ACK has `2.11.0` pre-installed and the latest helm version supported is `2.11.0` according to [the document](https://help.aliyun.com/document_detail/87014.html)